internal/core/infrastructure/daemon/grpc/service.go

```diff
@@ -0,0 +1,120 @@
+package grpcdaemon
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+
+	"github.com/tdex-network/tdex-daemon/pkg/macaroons"
+	"github.com/tdex-network/tdex-feeder/internal/core/ports"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"gopkg.in/macaroon.v2"
+
+	pboperator "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/operator"
+	pbunlocker "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/walletunlocker"
+	pbtypes "github.com/tdex-network/tdex-protobuf/generated/go/types"
+)
+
+var (
+	maxMsgRecvSize = grpc.MaxCallRecvMsgSize(1 * 1024 * 1024 * 200)
+)
+
+type service struct {
+	rpcAddress string
+
+	unlockerClient pbunlocker.WalletUnlockerClient
+	operatorClient pboperator.OperatorClient
+}
+
+func NewGRPCDaemon(
```

Comment:
`newTdexClient` more appropriate maybe ?

Reply:
I called it like this because it's possible on the daemon side to add other interfaces in the feature like for example HTTP, or JSON-RPC, etc.

Reply:
yeah its ok `newTdexGrpcClient` but still is not a daemon, but a client the data struct its returning